features/viewing-blog-posts.feature

```diff
@@ -1,30 +0,0 @@
-Feature: Viewing posts
-
-    Background:
-        Given there are posts:
-            | post_title      | post_content              | post_status | post_author |
-            | Just my article | The content of my article | publish     | admin       |
-            | My draft        | This is just a draft      | draft       | admin       |
-
-    Scenario: List my blog posts
-        Given I am on the homepage
-        Then I should see "Just my article"
-        And I should not see "My draft"
-
-    Scenario: Read a blog post
-        Given I am on the homepage
-        When I follow "Just my article"
-        Then I should see "Just my article"
-        And I should see "The content of my article"
-
-    Scenario: Viewing a single page
-        Given I am viewing a post:
```

Comment:
Sure - given today's discussion, I'll leave these in and will eventually move them over to wherever the permalink-related tests end up.